### ISSUE TITLE
[BUG] Sometimes after update, containers on the same network cannot reach each other

### ISSUE THREAD

**Issue by telnetdoogie** [telnetdoogie/synology-docker#9]
**Describe the bug**
if the `start_stop_status` script isn't successfully updated during the update, containers will fail to be able to reach each other due to a missing ipchains rule.

**To reproduce**
create two test containers:
```yml
services:
  test_alpine2:
    container_name: test_alpine2
    image: alpine:latest
    command: tail -F anything

  test_alpine:
    container_name: test_alpine
    image: alpine:latest
    command: tail -F anything
```
run the containers with `docker-compose up -d`
enter the console of `test_alpine`:
`docker-compose exec test_alpine /bin/sh -i`
attempt to ping container2 from container1:
`ping test_alpine`
the pings will timeout.

**Expected behavior**
after performing the above test, the pings should succeed.

**Additional context**
looking in the `/var/packages/ContainerManager/scripts/start-stop-status` there will be no entry `iptables -P FORWARD ACCEPT`.

**As a short term fix / validation that "yes this was indeed my issue" you can issue the command
`sudo iptables -P FORWARD ACCEPT` and the intra-container connectivity should immediately resolve.**

**Comment by telnetdoogie** [comment]
Should be fixed with 3a70aac

Will leave open until some validation from the community.

In the logs on update now you should see the line:

`Added missing IP forwarding configuration to /....filename...`

If the line is already there and doesn't need to be added during update, you should see a statement as such.

**Comment by telnetdoogie** [comment]
I have added a 'quick fix' for this. [fix_ipforward.sh](https://github.com/telnetdoogie/synology-docker/blob/main/fix_ipforward.sh) can be run as root and it should fix an upgrade where this was missed. It will add the entry to the `start-stop-status` script and immediately add the FORWARD policy - no docker restart or additional update needed:

```
sudo ./fix_upforward.sh
```
The quick fix will only work on DSM 7.2+
